obal/data/roles/diff_package/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+- block:
+    - name: 'Include brew package diff'
```

Comment:
Why not

```yaml
   - name: 'Include {{ diff_package_type }} package diff'
     include_tasks: "{{ diff_package_type }}.yml"
     tags:
       - always
```

Maybe with a `when: diff_package_type in ['brew', 'copr']` to allow non-diffed repos?